Ontologies.Mappings/src/Mappings/v1/Mapped/willow_v1_dtdlv2_mapped.json

```diff
@@ -530,7 +530,7 @@
     },
     {
       "InputDtmi": "dtmi:com:willowinc:CoilEnteringWaterTemperatureSensor;1",
-      "OutputDtmi": "dtmi:org:brickschema:schema:Brick:Entering_Water_Temperature_Sensor;1"
+      "OutputDtmi": "dtmi:mapped:core:Heat_Exchanger_Return_Water_Temperature_Sensor;1"
```

Comment:
The definition of Heat Exchanger is more limited than Coil so we should stick to the previous one.